content/en/monitors/downtimes/_index.md

```diff
@@ -164,3 +201,4 @@ Datadog can proactively mute monitors related to the manual shutdown of certain
 [16]: /integrations/amazon_ec2/#ec2-automuting
 [17]: /integrations/google_compute_engine/#gce-automuting
 [18]: /integrations/azure_vm/#automuting-monitors
+[19]: /logs/explorer/search_syntax/
```

Comment:
Would this link make more sense here or are these different?
https://docs.datadoghq.com/monitors/manage/search/